### DISCUSSION TITLE
Debate sobre el uso correcto de siglas y acrónimos en español según el PR #837

### DISCUSSION THREAD

**Discussion by NGHdeveloper** [reactjs/es.reactjs.org#839] [Q&A]
## Uso correcto de siglas y acrónimos en español

Abro esta discusión para debatir un tema muy interesante que se suscitó en el PR #837, y que realmente vale la pena aclarar, tanto para dicho PR como para la comunidad en general de traductores y comunicadores, sepan disculpar lo extenso y los posibles errores de ortografía y gramática (trato de ser cuidadoso en ello), pero es necesario tener los conceptos claros para el uso de nuestra lengua, mas aún cuando se trata de documentación. (según mi formación).

Paso a exponer mi, punto de vista:

La [RAE](https://www.rae.es/la-institucion), es la institución cuya función es elaborar normas para el empleo correcto de la lengua española (tanto hablada como escrita), como toda norma uno tiene la libertad de adoptarla o no, pero hay que tener en cuenta que 46 académicos bregan por mantener la historia, las formas y el criterio del uso de la lengua española (esto excede ampliamente mis conocimientos) y, por lo tanto, particularmente y por respeto a dichos académicos, me atengo firmemente al cumplimiento de las mismas.

En el PR #837, se exponen dos visiones muy interesantes de la utilización de la lengua escrita, una de las cuales adjunta dos enlaces iniciadores de este debate, (ver [Las abreviaciones | TeXnia](https://www.texnia.com/abbr.html) y [Guía de estilo - El proyecto MDN | MDN](https://developer.mozilla.org/es/docs/MDN/Writing_guidelines/Writing_style_guide#plurales)), ambos enlaces observando el uso de siglas y abreviaturas, desde mi punto de vista ambos son erróneos, el primero porque confunde conceptos del plural perfectamente definidos por la RAE para: [abreviaturas](https://www.rae.es/dpd/abreviatura) en el punto 7 y su [Apéndice 2. Lista de abreviaturas](https://www.rae.es/dpd/ayuda/abreviaturas) en el punto 3, [siglas](https://www.rae.es/dpd/sigla) en su punto 5 y [acrónimos](https://www.rae.es/dpd/acr%C3%B3nimo) en su punto 1.3, la página comienza definiendo las abreviaturas, luego especializa las abreviaturas y las clasifica en dos grupos y cito: “de fronteras algo difusas”, abreviaturas y siglas, y luego introduce a los acrónimos dentro de las siglas y al final dice y cito: 
“esas clasificaciones lejos de aclarar la situación la hacen aún más compleja”, luego indica, y cito: “La sigla es un tipo especial de abreviatura, sin que esté claro en qué se diferencia una de otra. Originalmente, las siglas (o lo que ahora entendemos por siglas) eran básicamente abreviaturas de nombres propios, por lo que se componían con mayúsculas.
Como no siempre eran pronunciables y había que deletrearlas, se amplió la forma de crear siglas mediante los acrónimos, que no solo tomaban las primeras letras de cada palabra sino también alguna más intermedia para poder formar algo pronunciable: así, en lugar de RNFE tenemos Renfe. Aunque se suele caracterizar el acrónimo por la inclusión letras intermedias, en realidad su razón de ser es permitir la lectura oral; así se define en francés e inglés y así debería definirse en español”, y vuelvo a citar el final…:
“así se define en francés e inglés y así debería definirse en español”….
Me parece que con esto último se excedió en sobremodo, puesto que interpreto que les está “sugiriendo” a los 46 académicos de la RAE como deben definir las normas ¡que ya están definidas!, y cuyos enlaces ya presenté más arriba, ¡quiero creer sinceramente que no las entendió!.
Luego dice y cito: “Las abreviaturas suelen formar el plural añadiendo s o es: Dirs., núms., Dres.”. Por supuesto que sí… en español….
Y termina con esta genialidad y cito:
“En los últimos años las abreviaturas y las siglas han dado un giro copernicano y es tiempo de reconsiderar las reglas. Yo escribo CDs y TVs, pues me parece un plural gráfico conciso, díafano, legible y reconocible, que se corresponde con su plural fonético y semántico. El hecho de que a los ingleses se les haya ocurrido antes no es razón suficiente para rechazarlo, y mucho menos para preferir un evidente galicismo. Más bien al contrario, en español las abreviaturas han tenido siempre plural”, ¿....?, ¡por Dios!, ¡cuanta liviandad!, ¡que facilidad para aseverar una falsedad!.
Ensalada completa, confusión absoluta, abreviaturas con siglas, siglas con acrónimos, acrónimos con abreviaturas, pienso sinceramente que la RAE los definió claramente en todos sus casos para el plural y la definición es la misma, no adopta el anglicismo de la letra s al final, con y sin apóstrofe ni comillas simples. Y si este señor Don [Javier Bezos](https://www.linkedin.com/in/jbezos/), el cual en [Linkedin](https://www.linkedin.com/in/jbezos/) dice realizar Ortotipografía, consultoría y Wikilengua en FundéuRAE (aún sin presentar ni estudios ni diplomas e incluso autodenominándose de “naturaleza autodidacta”), quiere escribir CDs o TVs o Matusalen's o mejor aún McDonald'ss (véase la doble s al final de la palabra) allá él, le recomiendo el uso correcto de las comas en la escritura (ademas de otros errores ortográficos) y que se postule para formar parte de los [cuarenta y seis académicos](https://www.rae.es/academicos/academicos-de-numero) que conforman la RAE (que realmente es quien asesora a FundéuRAE), porque no lo veo en el listado de la [Relación actual](https://www.rae.es/relacion-actual), pero al menos tiene una foto en [FundéuRAE](https://www.fundeu.es/noticia/defienden-la-profesion-de-corrector-frente-a-su-desaparicion-en-los-medios/javier-bezos-2/) que no dice nada, pero que el URL indica como "defienden-la-profesion-de-corrector-frente-a-su-desaparicion-en-los-medios" y un [link](https://www.fundeu.es/Sobrefundeubbva/quienes-somos/javier-bezos-lopez/javier-bezos-lopez/), que conduce a un error 404. Lo que si es verídico y comprobable es que actuó como colaborador en el [Manual del español urgente](https://www.fundeu.es/noticia/vuelve-el-manual-de-espanol-urgente-la-referencia-para-escribir-en-espanol/) en su reedición del 2015 cuyo autor es Fundéu, coordinada por él y editada por Debate, mas allá de eso, no encuentro ninguna obra académica de su autoría, que le dé autoridad para poner en tela de juicio nada más, ni nada menos, que a la RAE, y para terminar de desbaratar lo payasezco de este personaje agrego el enlace a la página de [siglas y acrónimos, claves de redacción](https://www.fundeu.es/recomendacion/siglas-y-acronimos-claves-de-redaccion/) de la misma FundéuRAE que dice asesorar.
Lamentablemente en la Web existe mucho de este tipo de confusiones, precisamente por no consultar a las fuentes.

En español… no se coloca s al final en abreviaturas a excepción de dos casos concisos y para palabras en español y cito a la RAE: 

"7. Plural. Si la abreviatura corresponde a una forma verbal o a una expresión oracional, para el plural se usa la misma forma que para el singular: cp. vale como abreviatura de compárese y de compárense; D. E. P. puede abreviar tanto Descanse en paz como Descansen en paz. Si la abreviatura corresponde a sustantivos o adjetivos con variación de número, el plural depende de su método de formación:

a) Si la abreviatura se obtuvo por truncamiento y, por tanto, termina en consonante, se añade -s: págs. por páginas, máxs. por máximos, perss. por personas. Algunas abreviaturas convencionales, no obstante, presentan plurales irregulares, como el de cent. (centavo, centésimo) y cént. (céntimo), que es cts., y no ⊗‍cents. ni cénts. (no debe confundirse la abreviatura cent. con la palabra cent, cuyo plural sí es cents; → [cent](https://www.rae.es/dpd/cent)). En abreviaturas formadas por una sola letra, obtenidas por truncamiento extremo, el plural se expresa duplicando esta: ss. por siguientes, EE. UU. por Estados Unidos.

b) Si la abreviatura se obtuvo por contracción y conserva el final de la palabra abreviada, se aplican las reglas generales de formación del plural y se añade -s o -es según sea la terminación (→ [plural](https://www.rae.es/dpd/plural), [1](https://www.rae.es/dpd/plural#S1590507321314821658)): dptos. o deptos. por departamentos, admones. por administraciones. Como excepción, Ud. (usted) y ud. (unidad) forman el plural en -s: Uds. (ustedes) y uds. (unidades). El plural de las abreviaturas con letras voladas debe representarse con este mismo tipo de letras: n.os por números. Las abreviaturas que no conservan la parte final de la palabra abreviada forman su plural añadiendo -s: mss. (por manuscritos), tlfs. (por teléfonos)".

En español... no se coloca la s final al plural en siglas en la lengua escrita (en lengua oral se pluralizan con normalidad), se recomienda que vayan acompañadas en lo posible de algún determinante que así lo indique, y de la RAE cito: 

"5. Plural. Aunque en la lengua oral las siglas pluralizan con normalidad ([oenejés] = 'organizaciones no gubernamentales'; [pesés, pezés] = ‘computadoras personales’), se mantienen invariables en la escritura. Para evidenciar su uso en plural, se recomienda que vayan acompañadas en lo posible de algún determinante que así lo indique: las ONG, varios CD. Se debe evitar, por tanto, el recurso, tomado del inglés, de añadir al final de la sigla una s minúscula, precedida o no de apóstrofo: ⊗‍ONGs, ⊗‍CD's. Solo los [acrónimo](https://www.rae.es/dpd/acr%C3%B3nimo)s lexicalizados, al ser palabras plenas a todos los efectos, pluralizan también en la escritura (→ acrónimo, [1](https://www.rae.es/dpd/acr%C3%B3nimo#S1590507317457446856) y [1.3](https://www.rae.es/dpd/acr%C3%B3nimo#S1590507317464736095)): ovnis (pl. del acrónimo de objeto volador no identificado), tacs (pl. del acrónimo de tomografía axial computarizada)".

Para el caso de los acrónimos para la RAE cito: 

"1.3. Plural. Si mantienen su condición de siglas y siguen escribiéndose enteramente con mayúsculas, son gráficamente invariables en plural: Aumento de los ERE en 2013 (→ [sigla](https://www.rae.es/dpd/sigla), [5](https://www.rae.es/dpd/sigla#S1590507322106881458)). En cambio, una vez lexicalizados, los acrónimos hacen el [plural](https://www.rae.es/dpd/plural) siguiendo las reglas generales de su formación en español (→ plural, [1](https://www.rae.es/dpd/plural#S1590507321314821658)): ovnis, mires (plural de mir, acrónimo de médico interno residente), pymes".

En español, no es lo mismo: una abreviatura, una sigla y un acrónimo, ni se tratan de igual manera que en el idioma inglés.

El segundo caso tomado de [Guía de estilo - El proyecto MDN | MDN](https://developer.mozilla.org/es/docs/MDN/Writing_guidelines/Writing_style_guide#plurales) es una traducción errónea y directa del inglés al español (proyecto que a futuro intentaré corregir) véase en [MDN en inglés abbreviations and acronyms / plurals](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#abbreviations_and_acronyms).

Este es mi humilde punto de vista, quedo abierto al debate.

**Comment by carburo** [comment]
Comprendo tu punto de vista, solo quisiera dejar algunas precisiones:
Estoy al tanto de la regla de la Academia en cuanto a los plurales de las siglas. Al mismo tiempo me parecen cuestionables en cuanto la propia Academia reconoce que las siglas se pluralizan como cualquier otra palabra en la lengua oral y luego establecen una regla distinta para el lenguaje escrito. Esto quedaría en el campo de lo anecdótico si al mismo tiempo no generara ambigüedades que luego la Academia recomienda resolver añadiendo artículos innecesarios, cuando bien podría optarse por reflejar la lengua oral en el lenguaje escrito. He abordado el tema porque a diferencia de muchos textos literarios las documentaciones técnicas como esta sí están plagados de siglas, por lo que para nosotros no es un tema menor. Al mismo tiempo no es algo que me quite el sueño por lo que voy sin ningún problema con lo que se decida aquí por los demás.

**Reply by NGHdeveloper** [reply]
Por eso es que precisamente la RAE define normas para la lengua oral y para la norma escrita, porque considera que no se debe escribir como se habla, la lengua escrita es "por mucho" mas estricta que la lengua oral, eso la RAE lo tiene cuenta por el tema de la diversidad en el habla y es mas permisiva, pero dicta normas mas estrictas en la escritura para que no haya deformaciones groseras en la lengua en general, y a medida que el volumen léxico aumenta, lexicaliza las nuevas palabras (o las clasifica como desusado/a). Esto es mas lógico que flexibilizar ambas, porque de ser así la RAE no cumpliría con su misión principal que es, y cito: "velar por que los cambios que experimente la lengua española en su constante adaptación a las necesidades de sus hablantes no quiebren la esencial unidad que mantiene en todo el ámbito hispánico, según establece el artículo primero de sus [actuales estatutos](https://www.rae.es/sites/default/files/2021-02/Estatutos%20y%20reglamento_2014_19_2_2021.pdf)", que constan de 133 páginas, son extensos, pero interesantes y fructíferos.

**Reply by NGHdeveloper** [reply]
Disculpame, me olvidarte de dar la explicación (que yo entendí) de la pluralización de las siglas en la lengua oral, se refiere a las siglas formadas en la lengua oral española ejemplo [pesés, pezés] = ‘computadoras personales’), porque PC es una sigla en lengua inglesa que no existe en nuestro diccionario (no está lexicalizada), deberíamos referirnos a ella como computadora personal.
¡Gracias por leer!.

**Comment by carburo** [comment]
Difícilmente lleguemos a un consenso en algo que es una cuestión subjetiva. Reconozco la importancia de un estándar para la lengua, pero al mismo tiempo creo que la Academia puede llegar a ser anticuada y excesivamente conservadora. Aunque personalmente prefiero la otra forma no me opongo a este cambio. En cualquier caso creo que sería importante añadir una sección en la guía de estilo, porque como hemos visto la mayoría de los contribuidores utilizan la forma «incorrecta». Si lo deseas se puede hacer el cambio desde ahora o esperar un tiempo por alguna opinión adicional.

**Reply by NGHdeveloper** [reply]
¡Muy buen debate!, constructivo y respetuoso, como debe ser, esperemos un tiempo, para ello iniciaré una votación para que la comunidad de su opinión en forma de voto, si te parece bien.

**Comment by mateoguzmana** [comment]
> Reconozco la importancia de un estándar para la lengua, pero al mismo tiempo creo que la Academia puede llegar a ser anticuada y excesivamente conservadora

Comparto la opinión de @carburo. El tema es bastante subjetivo y no tenemos que ser tan estrictos con esto siempre y cuando mantengamos la consistencia.

Con respecto a los estándares de otras comunidades, independientemente de que sean correctos o no, estoy seguro de que han habido muchas personas que han pensado, debatido y trabajado mucho en el tema antes, y creo que son una buena referencia. Algún motivo habrá detrás de ello :)

Dicho esto, tampoco es algo a lo que personalmente le doy mucha importancia. Si es algo que no tiene que cambiar masivamente toda la forma de escribir a lo largo de todas las páginas, me parece bien. De lo contrario, mantener la consistencia actual tampoco está mal.

**Comment by Byron2016** [comment]
Siempre se puede enviar un correo a la RAE y solicitarles que se pueda escribir IDs, ellos consideran esas cosas. Desde mi perspectiva si la oración está en plural, debería agregarse la "s" a la abreviación, incluso si leen `Estamos revisando LOS ID` se produce un **"que pasó acá en el cerebro"** 🤔 que no se da cuando se ha escrito `Estamos revisando LOS IDs` 🙂

**Comment by jbezos** [comment]
@NGHdeveloper 

Ya han pasado un par de años, pero acabo de ver este mensaje. Estaba dudando en si entrar o no, pero «este señor» ha decidido hacerlo, por alusiones.🙂

La norma ortográfica la dicta la RAE, es obvio. A este «[payasesco](https://dle.rae.es/payasesco) personaje» nunca le verás decir que las normas ortográficas generales no se deben seguir (sí, soy un poco 🤡, y a mucha honra). Bueno, a veces sí lo ha dicho, pero como opción personal. Las normas son opinables e, incluso, hay académicos que discrepan. 

En cuanto a tus argumentaciones _[ad hominem](https://www.rae.es/dpd/ad%20hominem)_, es verdad que no doy mis diplomas en LinkedIn, pero por una razón: para mí lo que pudiera aprender en su día encerrado en una clase no es lo relevante, sino lo que puedo transmitir a los demás de todo lo que he aprendido en mis años de experiencia, tanto en mi faceta vinculada a la FundéuRAE y la RAE como en la personal. Y en eso se centra mi perfil, que tal vez deberías leer completo, aunque eso tampoco me daría más la razón.

Mi opinión con relación a este debate es esta: yo seguiría lo que dice la RAE. Las [heterografías](https://www.wikilengua.org/index.php/Heterograf%C3%ADa) existen (y aquí soy heterógrafo), pero no me parecen adecuadas en un manual oficial. Repito, es mi opinión.

Un último apunte.

> “[...] Aunque se suele caracterizar el acrónimo por la inclusión letras intermedias, en realidad su razón de ser es permitir la lectura oral; así se define en francés e inglés y así debería definirse en español” [...].
Me parece que con esto último se excedió [[...] sobremodo](https://www.rae.es/dpd/sobremodo), puesto que interpreto que les está “sugiriendo” a los 46 académicos de la RAE [...]

Interpretas del revés. No digo en absoluto que el diccionario de la RAE lo defina con las letras intermedias. Me refiero a algunos manuales de estilo que, precisamente, no siguen la definición académica. En cualquier caso, incluso si fuera de la RAE no cambiaría nada, porque seguiría expresando mis discrepancias si considero que con ello puedo contribuir a mejorar las normas (o no, ¿quién sabe?).

**Reply by jbezos** [reply]
@NGHdeveloper Una observación con relación a la nomenclatura. No te falta razón en que, hoy, resulta imprecisa. El artículo original data de 2005, es decir, de antes de que la ortografía del 2010 fijara definitivamente la clasificación oficial, que antes era algo difusa (por ejemplo, en el DLE de 1992 la sigla todavía era un tipo de abreviatura y la ortografía de 1999 trataba las siglas en el capítulo de las abreviaturas). He hecho retoques menores posteriores, pero la mayoría del contenido es el original. Tal vez sea momento de actualizarlo de verdad.
